ui/src/app/pages/profile/profile-component.tsx

```diff
@@ -575,7 +576,7 @@ export const ProfileComponent = fp.flow(
                   firstSignInTime={profile.firstSignInTime}
                   onClick={() =>
                     enableUpdatedDemographicSurvey
-                      ? this.props.navigateByUrl('/demographic-survey')
+                      ? this.props.navigateByUrl(DEMOGRAPHIC_SURVEY_V2_PATH)
```

Comment:
nice.  Can this replacement be done elsewhere too?

Reply:
It was missing just from router, apart from that not sure if DEMOGRAPHIC_SURVEY_V2_PATH being used else where